spec/controllers/posts_controller_spec.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe PostsController, type: :controller do
```

Comment:
How about adding some integration tests in here for all CRUD operations? The tests defined in here are only **R**ead operations on `:new` and `:index` and `:show`. It'd be nice to include tests for Create, Update and Delete. When doing update and delete, test that only the post creator can make an edit or delete his own post and also test the opposite (that you can't edit or delete a post you haven't created).

Doing these tests now will help you out a lot while developing your app and give you less headaches later on.

[`OPTIONAL`]